pkgs/applications/misc/auto-multiple-choice/default.nix

```diff
@@ -137,6 +136,11 @@ stdenv.mkDerivation rec {
     XMLWriter
   ]);
 
+  passthru = {
+    tlType = "run";
+    pkgs = [ finalAttrs.finalPackage ];
```

Comment:
If this is the new standard, would you mind updating the docs:
https://github.com/NixOS/nixpkgs/blob/2dfe04b2d2373d552d782ac677b9348fd3bf66e4/doc/languages-frameworks/texlive.section.md?plain=1#L53

Reply:
I have updated the page, does it look ok?

Reply:
Looks very good!